tox.ini

```diff
@@ -5,12 +5,13 @@ envlist = {py35,py36,py37}-django{21,22},{py36,py37,py38}-django{30},lint
 [travis]
 python =
   3.5: py35
-  3.6: py36
+  3.6: py36, lint
   3.7: py37
   3.8: py38
 
 [travis:env]
 DJANGO =
+  2.1: django21
```

Comment:
Django 2.1 is EOL. So drop this line?

Reply:
It's not actually that line that triggers the build, that's sorted in `.travis.yml`

I had to add it to make it explicit that the py36-django21 environment should not include `lint`. It's a bit hard to explain, but it's here: https://tox-travis.readthedocs.io/en/stable/envlist.html#advanced-configuration